metadata.xml

```diff
@@ -0,0 +1,2 @@
+<dc:rights>Copyright 2013, erciccione</dc:rights>
```

Comment:
This guide is not copyrighted or licensed, so this string can be left empy for now. I plan to add a license soon btw

Reply:
Okay, I wasn't sure and put it in as a place holder and example.